tests/tests/deployment.rs

```diff
@@ -1,17 +1,29 @@
-use fuel_core::chain_config::ChainConfig;
+use fuel_core::chain_config::{
+    ChainConfig,
+    StateConfig,
+};
 
 #[test_case::test_case(
-    include_str!("../../deployment/scripts/chainspec/beta_chainspec.json"); 
+    include_str!("../../deployment/scripts/chainspec/beta_chainparams_spec.json"),
+    include_str!("../../deployment/scripts/chainspec/beta_chainstate_spec.json");
     "Beta chainconfig"
 )]
 #[test_case::test_case(
-    include_str!("../../deployment/scripts/chainspec/dev_chainspec.json"); 
+    include_str!("../../deployment/scripts/chainspec/dev_chainparams_spec.json"),
+    include_str!("../../deployment/scripts/chainspec/dev_chainstate_spec.json");
     "Dev chainconfig"
 )]
-fn test_deployment_chainconfig(bytes: &str) {
-    let chain_config: ChainConfig =
-        serde_json::from_str(bytes).expect("Should be able to decode chain config");
-    let actual_bytes = serde_json::to_string_pretty(&chain_config)
+fn test_deployment_chainconfig(params_bytes: &str, state_bytes: &str) {
```

Comment:
The same here. We need to work with directories and use the building methods to `ChainConfig` and `StateConfig`.